modules/post-installation-adding-nutanix-failure-domains-control-planes.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-control-planes_{context}"]
+= Distributing control planes across failure domains
+
+You distribute control planes across Nutanix failure domains by modifying the control plane machine set custom resource (CR).
+
+.Prerequisites
+
+* You have configured the failure domains in the cluster's Infrastructure custom resource definition (CR).
```

Comment:
Remove "definition", should be "Infrastructure custom resource (CR)".

Reply:
Done.